sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeCoordinatorSuite.scala

```diff
@@ -278,6 +278,25 @@ class ExchangeCoordinatorSuite extends SparkFunSuite with BeforeAndAfterAll {
     try f(spark) finally spark.stop()
   }
 
+  def withSparkSession(pairs: (String, String)*)(f: SparkSession => Unit): Unit = {
```

Comment:
why do we need it? we can still call the old `withSparkSession` and set confs in the body.

Reply:
ok